GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -825,11 +823,22 @@ private void ShowLoading(bool sync = true)
             _loadingControlAsync.BringToFront();
         }
 
-        public void ForceRefreshRevisions()
+        /// <summary>
+        ///  Indicates whether the revision grid can be refreshed, i.e. it is not currently being refreshed
+        ///  or it is not in a middle of reconfiguration process guarded by <see cref="SuspendRefreshRevisions"/>
+        ///  and <see cref="ResumeRefreshRevisions"/>.
+        /// </summary>
+        public bool CanRefresh => !_isRefreshingRevisions && _updatingFilters == 0;
+
+        /// <summary>
+        ///  Queries git for the new set of revisions, and refreshed the grid.
```

Comment:
Rename is of course fine, will just change this wording.
```suggestion
        ///  Queries git for the new set of revisions and refreshes the grid.
```

OK to squash and merge?
I would like to get #9735 ready to merge, that is both improving startup a lot and seem to improve test stability.